census/estimator/preprocessing/preprocess.py

```diff
@@ -0,0 +1,89 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Defines the Beam pipeline to preprocess the data."""
+
+import csv
+import os
+import random
+
+import apache_beam as beam
+import tensorflow as tf
+
+from constants import constants
+
+
+class CsvFileSource(beam.io.filebasedsource.FileBasedSource):
+  """Beam source for CSV files."""
+
+  def __init__(self, file_pattern, column_names):
+    self._column_names = column_names
+    super(self.__class__, self).__init__(file_pattern)
+
+  def read_records(self, file_name, unused_range_tracker):
+    self._file = self.open_file(file_name)
+    reader = csv.reader(self._file)
+    for rec in reader:
+      res = {key: val for key, val in zip(self._column_names, rec)}
+      yield res
+
+
+def split_data(examples, train_fraction):
```

Comment:
nit: Args information, 
examples is it a pandas.Dataframe, a file? 
train_fraction is it a float ?

Reply:
Added comment.